wiki/Tournaments/ATS/2018/en.md

```diff
@@ -61,25 +61,22 @@ The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
 
 -----------------------
 
-## Match Schedule: Semifinals
+## Match Schedule: Finals
 
-### Saturday, 23 June 2018
+### Saturday, 30 June 2018
 
-| Group | Player A |  |  | Player B | Match Time (UTC) | Local Time A | Local Time B |
-| :-: | --: | --: | :-- | :-- | :-: | :-: | :-: |
-| LB | ayomikasa | ![][flag_ID] | ![][flag_MY] | ZethZ161 | **08:00 UTC** | 15:00 UTC +7 | 16:00 UTC +8 |
-| LB | Xin Leviathan | ![][flag_JP] | ![][flag_JP] | KILI3343 | **09:30 UTC** | 18:30 UTC +9 | 18:30 UTC +9 |
-| LB | Smallwu | ![][flag_TW] | ![][flag_ID] | Niko-nyan | **12:00 UTC** | 20:00 UTC +8 | 19:00 UTC +7 |
-| LB | TS_LewisLuo | ![][flag_TW] | ![][flag_MY] | Jerry | **15:00 UTC** | 23:00 UTC +8 | 23:00 UTC +8 |
+| Match Number | Group | Player A |  |  | Player B | Match Time (UTC) | Local Time A | Local Time B |
+| :-: | :-: | --: | --: | :-- | :-- | :-: | :-: | :-: |
+| #28 | UB | nyanmi-1828 | ![][flag_JP] | ![][flag_JP] | butter0414 | **07:00 UTC** | 16:00 UTC +9 | 16:00 UTC +9 |
+| #25 | LB | Sanawieck | ![][flag_JP] | ![][flag_TW] | Smallwu | **08:30 UTC** | 17:30 UTC +9 | 16:30 UTC +8 |
+| #26 | LB | qoot8123 | ![][flag_TW] | ![][flag_JP] | Xin Leviathan | **10:00 UTC** | 18:00 UTC +8 | 19:00 UTC +9 |
 
-### Sunday, 24 June 2018
+### Sunday, 1 July 2018
 
 | Group | Player A |  |  | Player B | Match Time (UTC) | Local Time A | Local Time B |
```

Comment:
Seems its missing the Match number, so the table is kind of messed up.
Error propagates to translated files.